rs/execution_environment/tests/subnet_size_test.rs

```diff
@@ -85,15 +85,33 @@ fn inc_instruction_cost(config: HypervisorConfig) -> u64 {
         MeteringType::None => 0,
     };
 
-    let cd = if let MeteringType::New = config.embedders_config.metering_type {
+    let cost_dirty = if let MeteringType::New = config.embedders_config.metering_type {
         ic_config::subnet_config::SchedulerConfig::application_subnet()
             .dirty_page_overhead
             .get()
     } else {
         0
     };
 
-    5 * cc + cs + cl + ca + 2 * ccall + csys + cd
+    let cost_default = 1; // There must be some other unknown operator in the function with a cost of 1.
```

Comment:
That's exactly the default function cost, see: https://dfinity.atlassian.net/browse/RUN-785